docs/ssma/sybase/project-settings-conversion-sybasetosql.md

```diff
@@ -21,10 +21,10 @@ The Conversion pane is available in the **Project Settings** and **Default Proje
 -   To specify settings for the current project, on the **Tools** menu, select **Project Settings**, click **General** at the bottom of the left pane, and then click **Conversion**.  
   
 ## Miscellaneous Options  
-**@@ERROR**  
+**\@\@ERROR**  
```

Comment:
Would it make more sense to convert these sections to 3-rd level headings? I think it will be logically correct and it won't mix with descriptions of the setting values. What do you think?